src/main/java/lotto/domain/LottoSummary.java

```diff
@@ -0,0 +1,48 @@
+package lotto.domain;
+
+import java.util.EnumMap;
+import java.util.List;
+
+public class LottoSummary {
+
+    private final EnumMap<LottoRank, Integer> rankCounts = new EnumMap<>(LottoRank.class);
```

Comment:
EnumMap 배우고 갑니다! 다만 변수 타입은 언제라도 변경될 수 있으니 다음과 같이 해도 좋을 것 같아요!
```suggestion
    private final Map<LottoRank, Integer> rankCounts = new EnumMap<>(LottoRank.class);
```